translate/storage/test_poheader.py

```diff
@@ -51,10 +51,9 @@ def test_update():
     d['Project-Id-Version'] = 'abc'
     d['POT-Creation-Date'] = 'now'
     d = poheader.update(d, add=True, Test='hello', Report_Msgid_Bugs_To='bugs@list.org')
-    assert d.keys()[0] == "Project-Id-Version"
-    assert d.keys()[1] == "Report-Msgid-Bugs-To"
-    assert d.keys()[2] == "POT-Creation-Date"
-    assert d.keys()[3] == "Test"
+    assert list(d.keys()) == [
+        "Project-Id-Version", "Report-Msgid-Bugs-To", "POT-Creation-Date", "Test",
```

Comment:
The last element is beyond the preferred maximum line length, so please wrap it.